### PR TITLE
feature: adds search guide german

### PR DESCRIPTION
**Pull request by mb-wali**
* adds a german template for the search guide.

### CODE REVIEW

**Comment by mb-wali** [comment]
## TODO
* take the latest version of the document which has the title translation.
* add german translated version instead
* add copyrights

File: invenio_config_tugraz/templates/invenio_app_rdm/help/search.de.html [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+    <p>
+      Die Ergebnisse entsprechen Datensätzen mit den <em>Begriffen</em> <code>open</code>
+      <em>oder</em> <code>science</code> in einem <em>beliebigen Feld</em>. Beachten Sie, dass die Abstammung angewendet wird,
+      so dass z. B. <code>science</code> auch mit
+      <code>sciences</code>übereinstimmt. Die Suchergebnisse werden nach einem Algorithmus geordnet,
+      der Ihre Suchbegriffe berücksichtigt.
+    </p>
+    <p>
+      Sie können das <em>Vorhandensein</em> beider Terme entweder mit dem
+      <code>+</code> oder dem <code>AND</code> Operator verlangen:
```
Comment (chriz-uniba) [inline invenio_config_tugraz/templates/invenio_app_rdm/help/search.de.html:34]: For me 'verlangen' sounds strange. Might it be 'erzwingen'?

File: invenio_config_tugraz/templates/invenio_app_rdm/help/search.de.html [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+    <h3>Feldsuche</h3>
+    <p>
+      <strong>Beispiel:</strong>
+      <a href="/search?page=1&amp;size=20&amp;q=metadata.title:open"
+        ><code>metadata.title:open</code></a
+      >
+    </p>
+    <p>
+      Die Ergebnisse stimmen mit Datensätzen überein, bei denen der <em>Begriff</em> <code>open</code> im
+      <em>Feld</em> <code>metadata.title</code>ist. enn Sie nach mehreren Begriffen im Titel suchen möchten,
```
Comment (chriz-uniba) [inline invenio_config_tugraz/templates/invenio_app_rdm/help/search.de.html:80]: 'enn' should be 'Wenn'?

File: invenio_config_tugraz/templates/invenio_app_rdm/help/search.de.html [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+    <p>
+      <strong>Beispiel:</strong>
+      <a href="/search?page=1&amp;size=20&amp;q=metadata.title:(open%20science%29"
+        ><code>metadata.title:(open science)</code></a
+      >
+    </p>
+    <p>
+      Die vollständige Liste der Felder, die Sie durchsuchen können, finden Sie in der Feldreferenz unten.
+    </p>
+    <h3>Kombinierte einfache, Phrasen- oder Feldsuche</h3>
```
Comment (chriz-uniba) [inline invenio_config_tugraz/templates/invenio_app_rdm/help/search.de.html:92]: Is the ',' (comma) right?

File: invenio_config_tugraz/templates/invenio_app_rdm/help/search.de.html [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+      als auch in der REST-API ist es möglich, die Ergebnisse zu sortieren nach:
+    </p>
+    <ul>
+      <li>Aktuellste</li>
+      <li>Beste Übereinstimmung</li>
+    </ul>
+    <h3>Reguläre Ausdrücke</h3>
+    <p>
+      Reguläre Ausdrücke sind eine mächtige Sprache für den Mustervergleich, mit der
+      man nach bestimmten Mustern in einem Feld suchen kann. Wenn wir zum Beispiel
```
Comment (chriz-uniba) [inline invenio_config_tugraz/templates/invenio_app_rdm/help/search.de.html:158]: Here we start to talk about 'wir' (we) while beforehand it was 'unpersonal' (one can) or 'Sie' (formal you). It feels a bit like a style break.

File: invenio_config_tugraz/templates/invenio_app_rdm/help/search.de.html [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+        ><code>_exists_:metadata.creators</code></a
+      >
+      (alle Datensätze mit metadata.creators)
+    </p>
+    <h3>Erweiterte Konzepte</h3>
+    <h4>Boosten</h4>
+    <p>
+      Sie können den Boost-Operator <code>^</code> verwenden, wenn ein Begriff relevanter ist als ein
+      anderer. Sie können z. B. nach allen Datensätzen suchen, die den Ausdruck
+      <em>open science</em> entweder <em>im title</em> oder im
```
Comment (chriz-uniba) [inline invenio_config_tugraz/templates/invenio_app_rdm/help/search.de.html:202]: Should the 'im' in 'im title' be emphazized? The next one (im description) is not.

File: invenio_config_tugraz/templates/invenio_app_rdm/help/search.de.html [hunk 6]
```diff
@@ -0,0 +1,257 @@
...
+      <em>description</em> feld enthalten, aber Datensätze mit dem Ausdruck im
```
Comment (chriz-uniba) [inline invenio_config_tugraz/templates/invenio_app_rdm/help/search.de.html:203]: 'feld' should be Feld?

File: invenio_config_tugraz/templates/invenio_app_rdm/help/search.de.html [hunk 7]
```diff
@@ -0,0 +1,257 @@
...
+    <p>
+      Mit dem Fuzzy-Operator <code>~</code>können Sie nach
+      Begriffen suchen, die Ihrem Suchbegriff ähnlich, aber nicht genau gleich sind.
+    </p>
+    <p>
+      <strong>Beispiel:</strong>
+      <a href="/search?page=1&amp;size=20&amp;q=oepn~"><code>oepn~</code></a>
+    </p>
+    <p>
+      Die Ergebnisse entsprechen Datensätzen mit ähnlichen Begriffen wie <code>oepn</code> die z. B.
```
Comment (chriz-uniba) [inline invenio_config_tugraz/templates/invenio_app_rdm/help/search.de.html:223]: I'm not sure whether the typo is confusing without explaining that it is a 'wanted' typo (oepn).

File: invenio_config_tugraz/templates/invenio_app_rdm/help/search.de.html [hunk 8]
```diff
@@ -0,0 +1,257 @@
...
+      auch auf <code>open</code> passen würden.
+    </p>
+    <h4>Näherungssuche</h4>
+    <p>
+      Eine Phrasensuche wie <code>"open science"</code> erwartet standardmäßig alle Begriffe in
```
Comment (chriz-uniba) [inline invenio_config_tugraz/templates/invenio_app_rdm/help/search.de.html:228]: Is this clear for non-tec-people that the 'quotes' (") are necessary?

File: invenio_config_tugraz/templates/invenio_app_rdm/help/search.de.html [hunk 9]
```diff
@@ -0,0 +1,257 @@
...
+    </p>
+    <p>
+      <strong>Beispiel:</strong>
+      <a href="/search?page=1&amp;size=20&amp;q=%22open%20science%22~5"
+        ><code>"open science"~5</code></a
+      >
+    </p>
+    <h4>Wildcards</h4>
+    <p>
+      Sie können Wildcards in Suchbegriffen verwenden, um ein einzelnes Zeichen (mit
```
Comment (chriz-uniba) [inline invenio_config_tugraz/templates/invenio_app_rdm/help/search.de.html:243]: This sentence is super hard to understand.
- 'Operator<code>?</code> operator'
- what means 'null' (either one character or null character)?
- ' Zeichen  (mit' two blanks between 'n' and '('.

**Pull request merged**